cypress/e2e/app/recommendations/filter-summary-records.cy.ts

```diff
@@ -155,12 +155,80 @@ describe("Recommendations", () => {
         );
         cy.get(".filters-drawer-container .mat-drawer-opened").should("exist");
         cy.wait(2000);
-        cy.get("td.cdk-column-designatedBody").each(($el, index, $list) => {
+        cy.get("td.cdk-column-designatedBody").each(($el) => {
           expect($el.text() === "1-AIIDWT").to.be.false;
         });
       });
     });
 
+    describe("Filter by GMC status", () => {
+      it("should display 'gmc status' selection list filters", () => {
+        initFilterPanel();
+        cy.get("[data-cy='selectionListgmcStatus']").should("exist");
+      });
+
+      it("should display 3 selection options in the 'gmc status' selection list labelled accordingly", () => {
+        const gmcStatusLabels = ["Approved", "Rejected", "Under Review"];
```

Comment:
Looking at where some or all of these are referenced, would it be worth defining these elsewhere or using values from `src/app/recommendations/constants.ts`?

Reply:
This is an interesting point. To what extent should your test code be independent from your code? As a tester I'm defining what I expect to see which I feel should be separate to what the programmer has defined.

Reply:
Yes.  We definitely want it to be independent of implementation specifics and we stop short of having a different git repo for tests.  An assumption: the test dependencies include the "implementation" dependencies too.  Where we have constants, they operate a lot like that "common" config.
This was the question anyhow.  Test isolation, as in tests that are independent of each other and therefore don't cause each other to break might also be a reason you want to define this multiple times across tests but I took these labels as "`const`ant" enough that if they change in one place, we'd need to change them everywhere.
Happy for you to resolve & continue conversation elsewhere if that's useful.